code/modules/jobs/job_types/followers.dm

```diff
@@ -345,24 +345,56 @@ Follower Volunteer
 		/obj/item/flashlight/seclite = 1,
 		/obj/item/storage/firstaid/ancient = 1,
 		/obj/item/stock_parts/cell/ammo/ec = 2,
+		/obj/item/melee/onehanded/knife/hunting=1,
+	)
+
+/datum/outfit/job/followers/f13followerscholar/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	ADD_TRAIT(H, TRAIT_TECHNOPHREAK, src)
+	ADD_TRAIT(H, TRAIT_RESEARCHER, src)
+	ADD_TRAIT(H, TRAIT_CYBERNETICIST_EXPERT, src)
+	ADD_TRAIT(H, TRAIT_CYBERNETICIST, src)
+
+/datum/outfit/loadout/engineer //Upgraded tools, RCD
+	name = "Followers Engineer"
+	belt = /obj/item/storage/belt/utility/full/engi
+	head =	/obj/item/clothing/head/hardhat
+	gloves =	/obj/item/clothing/gloves/color/yellow
+	backpack_contents = list(
+		/obj/item/construction/rcd = 1,
```

Comment:
No, thanks. They'd be the only group to get something like this roundstart, no?
Check how Engineer is done for NCR.

Reply:
RCDs themselves can be made in an autolathe, but, point! Replacing with metal and glass.